libbeat/template/field.go

```diff
@@ -11,18 +12,19 @@ var (
 )
 
 type Field struct {
-	Name           string `config:"name"`
-	Type           string `config:"type"`
-	Description    string `config:"description"`
-	Format         string `config:"format"`
-	ScalingFactor  int    `config:"scaling_factor"`
-	Fields         Fields `config:"fields"`
-	MultiFields    Fields `config:"multi_fields"`
-	ObjectType     string `config:"object_type"`
-	Enabled        *bool  `config:"enabled"`
-	Analyzer       string `config:"analyzer"`
-	SearchAnalyzer string `config:"search_analyzer"`
-	Norms          bool   `config:"norms"`
+	Name           string       `config:"name"`
+	Type           string       `config:"type"`
+	Description    string       `config:"description"`
+	Format         string       `config:"format"`
+	ScalingFactor  int          `config:"scaling_factor"`
+	Fields         Fields       `config:"fields"`
+	MultiFields    Fields       `config:"multi_fields"`
+	ObjectType     string       `config:"object_type"`
+	Enabled        *bool        `config:"enabled"`
+	Analyzer       string       `config:"analyzer"`
+	SearchAnalyzer string       `config:"search_analyzer"`
+	Norms          bool         `config:"norms"`
+	Dynamic        *dynamicType `config:"dynamic"`
```

Comment:
you don't really need a pointer here, as `Dynamic.value` will be `nil` if not set via `Unpack`.